docker-compose.dev.yml

```diff
@@ -267,6 +267,13 @@ services:
             file: docker-compose.base.yml
             service: localstack
 
+    seaweedfs:
+        extends:
+            file: docker-compose.base.yml
+            service: seaweedfs
+        volumes:
+            - './data/seaweedfs:/data'
```

Comment:
Was there a specific reason we avoided a named volume here? It’d simplify teardown and reduce local data accumulation across a lot of dev environments.